mdata/store_cassandra.go

```diff
@@ -362,9 +370,15 @@ func (o asc) Less(i, j int) bool { return o[i].sortKey < o[j].sortKey }
 
 func (c *CassandraStore) processReadQueue() {
 	for crr := range c.readQueue {
-		cassGetWaitDuration.Value(time.Since(crr.timestamp))
+		waitDuration := time.Since(crr.timestamp)
```

Comment:
I think we should set the crr's timestamps more consistently (wrt all other crr's from the same SearchTable call). concretely, instead of calling `time.Now()` in each query() call, we can just reuse `pre`. since the time in between each query call is negligible anyway.
this will help avoid a class of race conditions that is pretty rare I think, but might happen, where we omit here certain CRR's due to old, while processing others that were just under the threshold. if we omit one, we might as well omit all others from the same request, and by using the same timestamp in all, we can guarantee that all CRR's after an omitted one, will also be omitted.

(also it would perform a bit better by not getting the time over and over)